package.json

```diff
@@ -64,6 +64,7 @@
     "test": "npm run lint && mongodb-test-runner -t 60000 test/core test/unit test/functional",
     "coverage": "istanbul cover mongodb-test-runner -- -t 60000 test/core test/unit test/functional",
     "lint": "eslint lib test",
+    "lint-test-runner-poc": "eslint test-runner-poc/lib test-runner-poc/test",
```

Comment:
I think you can make this `eslint test-runner-poc` or you can add `test-runner-poc` to the line above.

Reply:
We are specifying the `lib` and `test` folders because we want to prevent the linter from running on all the node-modules.